addons/website_sale/static/src/js/website_sale.js

```diff
@@ -500,6 +505,15 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
         });
         this.$form.append($productNoVariantAttributeValues);
 
+        if (this.isBuyNow) {
+            var $expressCheckout = $('<input>', {
```

Comment:
```javascript
            $('<input>', {
                name: 'express',
                type: "hidden",
                value: true
            }).appendTo(this.$form);
```

idem for 2 upper adds.
Avoid to create useless variable when possible and without loose readability.